Catppuccin VS Themes/Catppuccin Frappé.vstheme

```diff
@@ -1,5 +1,5 @@
 <Themes>
-  <Theme Name="Catppuccin Frappé" GUID="{217075d3-6c1c-432d-9587-ca060f7229fe}" BaseGUID="{1ded0138-47ce-435e-84ef-9ec1f439b749}">
+  <Theme Name="Catppuccin Frappé (7)" GUID="{f215a980-7da4-45b3-8020-0785d4e29ee2}" BaseGUID="{1ded0138-47ce-435e-84ef-9ec1f439b749}">
```

Comment:
Theme GUID and names should not change.

Reply:
fixed - that's a weird behavior of the vs theming extension when you try to review manual changes by applying those directly from the designer editor...